src/cursor.rs

```diff
@@ -1,11 +1,10 @@
 use ffi;
 use statement::{State, Statement};
-use {Result, Value};
+use {Result, Row, Value};
 
 /// An iterator over rows.
 pub struct Cursor<'l> {
     state: Option<State>,
-    values: Option<Vec<Value>>,
```

Comment:
I'm removing this memory allocation optimization for now because Iterator needs to return an owned type anyway.